services/galley/test/integration/API/Teams/Feature.hs

```diff
@@ -678,7 +679,9 @@ testAllFeatures = do
             .= Public.TeamFeatureStatusNoConfigAndLockStatus
               TeamFeatureEnabled
               Public.Unlocked,
-          toS TeamFeatureValidateSAMLEmails .= Public.TeamFeatureStatusNoConfig TeamFeatureEnabled
+          toS TeamFeatureValidateSAMLEmails .= Public.TeamFeatureStatusNoConfig TeamFeatureEnabled,
+          toS TeamFeatureGuestLinks .= Public.TeamFeatureStatusNoConfigAndLockStatus TeamFeatureEnabled Public.Unlocked,
```

Comment:
Oops?  Oversight, right?  Can we do something straight-forward with `[minBound..]` here to catch these in the future?  (Not in this PR, pls.)

Reply:
I'll think about it